### ISSUE TITLE
Is there a way to use the interpreter in pm2-dev?

### ISSUE THREAD

**Issue by uyu423** [Unitech/pm2#3100]
I want to use `pm2-dev` instead of nodemon.
But I can not find an option to use the interpreter in pm2-dev.
For `nodemon`, you can use `nodemon --exec babel-node index.js`.
For `pm2`, you can use `pm2 start index.js --interpreter ./node_modules/.bin/babel-node`.
How can you use the interpreter in pm2-dev?

```
===============================================================================
--- PM2 REPORT (Fri Aug 18 2017 20:33:18 GMT+0900 (KST)) ----------------------
===============================================================================
--- Daemon -------------------------------------------------
pm2d version         : 2.6.1
node version         : 8.4.0
node path            : /usr/bin/pm2
argv                 : /usr/bin/nodejs,/usr/lib/node_modules/pm2/lib/Daemon.js
argv0                : node
user                 : yowu
uid                  : 1000
gid                  : 1000
uptime               : 0min
===============================================================================
--- CLI ----------------------------------------------------
local pm2            : 2.6.1
node version         : 8.4.0
node path            : /usr/bin/pm2
argv                 : /usr/bin/nodejs,/usr/bin/pm2,report
argv0                : node
user                 : yowu
uid                  : 1000
gid                  : 1000
===============================================================================
--- System info --------------------------------------------
arch                 : x64
platform             : linux
type                 : Linux
cpus                 : Intel(R) Pentium(R) CPU G4400 @ 3.30GHz
cpus nb              : 2
freemem              : 924635136
totalmem             : 8048590848
home                 : /home/yowu
===============================================================================
```

**Comment by uyu423** [comment]
I checked the `./bin/pm2` and `./lib/binaries/devCLI.js`, but the interpreter options for `pm2-dev` do not seem to be considered from the beginning.
I would have contributed if I understood the system of pm2, but I can not keep up with the complexity.
Somebody help me!

**Comment by uyu423** [comment]
@vmarchaud  Nice. Thank you. I truly love you.

**Comment by Unitech** [comment]
Fixed on recently published pm2 (2.7.0):

```bash
$ npm install pm2@latest -g
$ pm2 update
```

Let me know if there is any issue
